src/Daemon/BaseDaemon.cpp

```diff
@@ -395,12 +406,7 @@ class SignalListener : public Poco::Runnable
 #if defined(SANITIZER)
 extern "C" void __sanitizer_set_death_callback(void (*)());
 
-/// Sanitizers may not expect some function calls from death callback.
-/// Let's try to disable instrumentation to avoid possible issues.
-/// However, this callback may call other functions that are still instrumented.
-/// We can try [[clang::always_inline]] attribute for statements in future (available in clang-15)
-/// See https://github.com/google/sanitizers/issues/1543 and https://github.com/google/sanitizers/issues/1549.
-static DISABLE_SANITIZER_INSTRUMENTATION void sanitizerDeathCallback()
+static void sanitizerDeathCallback()
```

Comment:
How is this change connected to the current PR?

Reply:
This was probably by changed by the clang formatter. There is no relation with this PR. Changes submitted. Thanks!